_build/go.mod

```diff
@@ -416,3 +416,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/expr-lang/expr => github.com/expr-lang/expr v1.17.2
```

Comment:
Why replace and not a standard version bump?

Reply:
You're right, replace don't needed.